Puzzlepart/did
# Watch task can automatically start a localtunnel

### Your checklist for this pull request

### Review checklist

### Description
Watch task can automatically start a localtunnel using `process.env.LOCALTUNNEL_SUBDOMAIN`.

See comments on the code changes below.

### How to test
1. Check out locally with [gh](https://github.com/cli/cli)
2. Set `LOCALTUNNEL_SUBDOMAIN` in your `.env` file to e.g. `did591`
3. Run `npm run watch`
4. Ensure **https://did591.loca.lt** is up running
